### ISSUE TITLE
[FR] Support for using Ollama for AI

### ISSUE THREAD

**Issue by not-a-ethan** [AppFlowy-IO/AppFlowy#5374]
### Description

Add the ability to use Ollama for AI instead of only openAI.

### Impact

People that want to use AI features but want to use a self-hosted AI

**Comment by Aarplexan** [comment]
i agree with you it would be very convenient to be able to use our selfhoster Llama AI instead of OpenAI

**Comment by spj2401Dev** [comment]
Yea, that would fit in the AppFlowy Vision of owning your Data. 
They advertise owning your Data and stand for privacy and then they outsource all your Data to OpenAI.

**Comment by samyogdhital** [comment]
https://appflowy.io/privacy

See #Interactive Features section in this page.
  `We and other users of our Website or Services collect the information you submit or make available through these interactive features, including your name, email address, username and any personal information included in the contents, inputs, file uploads, or feedback you provide to these interactive features, as well as in the results or outputs generated from these interactive features (“User Content”). `

They are already using your data written in appflowy to their advantage. They are potentially training their model on these data.

**Comment by annieappflowy** [comment]
#4021

**Comment by annieappflowy** [comment]
Supported in the upcoming release (v0.8.7)
https://appflowy.com/guide/appflowy-local-ai-ollama
